internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -198,17 +198,17 @@ func convertCapacityProviders(a manifest.AdvancedCount) []*template.CapacityProv
 		CapacityProvider: capacityProviderFargateSpot,
 	})
 	rc := a.Range.RangeConfig
-	// Return if only spot is specifed as count
+	// Return if only spot is specified as count
 	if rc.SpotFrom == nil {
 		return cps
 	}
 	// Scaling with spot
 	spotFrom := aws.IntValue(rc.SpotFrom)
 	min := aws.IntValue(rc.Min)
-	// If spotFrom value is not equal to the autoscaling min, then
-	// the base value on the Fargate Capacity provider must be set
+	// If spotFrom value is greater than or equal to the autoscaling min, 
+	// then the base value on the Fargate capacity provider must be set
 	// to one less than spotFrom
-	if spotFrom > min {
+	if spotFrom >= min {
 		base := spotFrom - 1
```

Comment:
huh so you can you can have `spotFrom` and `min` be 0? 
and it doesn't error if the base ends up being -1?

According to the commit message, we get:
>min: 0 and spot_from: 0 --> 0 Fargate, 1 Spot 

I think this is pretty strange, I would have expected 0 Fargate and 0 Spot here.

Reply:
Side-note: should we make sure that `min`, `max` and `spot_from` is >= 0 here:
https://github.com/aws/copilot-cli/blob/5781c535bc78e5e3a544b855d930256562e52b63/internal/pkg/manifest/validate.go#L1077

Reply:
![Screen Shot 2022-11-16 at 3 33 02 PM](https://user-images.githubusercontent.com/60631893/202317705-a0688c04-4a3f-4082-b986-8a30fa7f5d0e.png)

Reply:
this has got to be a bug lol 😂 like what could -1 base mean conceptually